python/ray/train/_internal/state/schema.py

```diff
@@ -45,11 +46,33 @@ class TrainRunInfo(BaseModel):
     datasets: List[TrainDatasetInfo] = Field(
         description="A List of dataset info for this Train run."
     )
-    start_time: float = Field(
+    start_time_ms: int = Field(
         description="The UNIX timestamp of the start time of this Train run."
     )
 
 
+@DeveloperAPI
+class TrainRunInfoWithDetails(BaseModel):
```

Comment:
Do we need a separate class for this? 

If yes, can we just have this subclass `TrainRunInfo` so we don't have to duplicate keys?

Reply:
good idea. Can't believe I didn't think of that.